### PR TITLE
Fix WL-2953

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: web-tool/tool/src/webapp/vm/web/chef_iframe-customize.vm [hunk 1]
```diff
@@ -66,7 +124,9 @@
 	#if ($showPopup)
 		<p class ="checkbox labelindnt">
 				<input type="checkbox" value="true" name="popup" id="opentrue" #if ($popup) checked="checked" #end  class="indnt1" />
-				<label for="opentrue">$tlang.getString("gen.info.pop")</label>
+				<!-- hidden to force a submission when the checkbox is disabled -->
+				<input type="hidden" name="popupHidden" id="opentrueHidden" value="true" disabled="true" />
+				<label for="opentrue">$tlang.getString("gen.info.pop") <span style="display: none; font-style: italic;" id="popupDisabledMessage">Must be enabled for insecure content (http).</span></label>
```
Comment (buckett) [inline web-tool/tool/src/webapp/vm/web/chef_iframe-customize.vm]: Best to have text come from a properties file.

**Comment by buckett** [comment]
I'd be tempted to use jQuery for this one as it's HTML we (service provider) control.

**Comment by ghost** [comment]
So would I. I guess it depends how much more work / maintenance we can expect to do for this bug since we've already got the code in vanilla javascript.

It could certainly be a good start to change the event handlers to jQuery.

File: web-tool/tool/src/webapp/vm/web/chef_iframe-customize.vm [hunk 2]
```diff
@@ -2,6 +2,64 @@
...
+	urlStartsWithHttp = function() {
+		if(urlInput && urlInput.value) {
+			return urlInput.value.match(/^http:/);
+		} else {
+			return false;
+		}
+	}
+
+	checkPopupIfProtocolIsHttp = function() {
+		popupCheckbox = window.self.document.getElementById('opentrue');
```
Comment (buckett) [inline web-tool/tool/src/webapp/vm/web/chef_iframe-customize.vm]: These are polluting the global scope, could you limit their definition with var?

**Comment by ghost** [comment]
Done

**Pull request merged**